raiden/tests/integration/network/transport/test_matrix_transport.py

```diff
@@ -151,7 +155,7 @@ def ping_pong_message_success(transport0, transport1):
     return all_messages_received
 
 
-def is_reachable(transport: MatrixTransport, address: Address) -> bool:
+def is_reaceqhable(transport: MatrixTransport, address: Address) -> bool:
```

Comment:
typo here, be careful cause this function is used on: 

- test_matrix_invite_private_room_unhappy_case2
- test_matrix_invite_private_room_unhappy_case3
- test_matrix_user-roaming

Reply:
Yes good catch, i don't know how the complete set of tests pass with this :S

Reply:
Make sense, i removed all the usages so it's ok. I fixed this typo